### PR TITLE
Support automated tests

### PR DESCRIPTION
**Pull request by kaibolay**

### CODE REVIEW

File: lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb [hunk 1]
```diff
@@ -403,6 +498,39 @@ def self.available_options
...
 
+          # Release Testing
+          FastlaneCore::ConfigItem.new(key: :test_devices,
+                                       env_name: "FIREBASEAPPDISTRO_TEST_DEVICES",
+                                       description: "Comma-separated list of devices to run automated tests on, in the format '<model id>|<os version id>|<locale>|<orientation>' (e.g. 'shiba|34|en|portrait'). Run 'gcloud firebase test android|ios models list' to see available devices",
+                                       optional: true,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :test_devices_file,
+                                       env_name: "FIREBASEAPPDISTRO_TEST_DEVICES_FILE",
+                                       description: "Comma-separated list of devices to run automated tests on, in the format '<model id>|<os version id>|<locale>|<orientation>' (e.g. 'shiba|34|en|portrait'). Run 'gcloud firebase test andoid|ios models list' to see available devices",
```
Comment (lfkellogg) [inline lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb:509]: "Path to file containing..."

**Review by lfkellogg — APPROVED** [review]
Awesome!

File: lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb [hunk 2]
```diff
@@ -403,6 +510,39 @@ def self.available_options
                                        optional: true,
                                        type: String),
 
+          # Release Testing
+          FastlaneCore::ConfigItem.new(key: :test_devices,
+                                       env_name: "FIREBASEAPPDISTRO_TEST_DEVICES",
+                                       description: "Comma-separated list of devices to run automated tests on, in the format '<model id>|<os version id>|<locale>|<orientation>' (e.g. 'shiba|34|en|portrait'). Run 'gcloud firebase test android|ios models list' to see available devices",
```
Comment (kaibolay) [inline lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb:516]: this need to be updated

File: lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb [hunk 3]
```diff
@@ -403,6 +510,39 @@ def self.available_options
...
+                                       optional: true,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :test_devices_file,
+                                       env_name: "FIREBASEAPPDISTRO_TEST_DEVICES_FILE",
+                                       description: "Path to file containing comma-separated list of devices to run automated tests on, in the format '<model id>|<os version id>|<locale>|<orientation>' (e.g. 'shiba|34|en|portrait'). Run 'gcloud firebase test andoid|ios models list' to see available devices",
```
Comment (kaibolay) [inline lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb:521]: this need to be updated

Comment (tagboola) [reply lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb:521]: Done.

File: lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb (hunk 2 above)
Comment (tagboola) [reply lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb:516]: Done.

**Pull request merged**